src/pages/nft/collection/_collectionId/index.vue

```diff
@@ -226,6 +248,10 @@ export default {
         };
         logPurchaseFlowEvent(this, 'add_to_cart', purchaseEventParams);
         logPurchaseFlowEvent(this, 'begin_checkout', purchaseEventParams);
+        const customPriceInDecimal =
+          this.customPrice > -1
+            ? this.formatCustomPrice(this.customPrice, this.collectionPrice)
+            : undefined;
```

Comment:
I think we can assume `customPrice === 0` means no custom price (`undefined`) too

Reply:
[🎨 Update custom initial price](https://github.com/likecoin/liker-land/pull/1554/commits/144c735d680c87129eaad32f565c015356435b59)